src/main/template/operation.handlebars

```diff
@@ -4,15 +4,17 @@
       <div class='heading'>
         <h3>
           <span class='http_method'>
-          <a href='#!/{{encodedParentId}}/{{nickname}}' class="toggleOperation">+</a>
+          <a href='#{{parentId}}_{{nickname}}' data-href='#!/{{encodedParentId}}/{{nickname}}' class="toggleOperation">+</a>
```

Comment:
Ah. Well one of those is `data-href`, but i don't see where that's being used. What's this for @nickbristow ?